components/infoModal/ReferenceComments.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { Product } from "@/lib/types/types";
+import { FaCheck } from "react-icons/fa";
+
+const ReferenceComments = ({ currentProduct }: any) => {
+  return (
+    <div className="h-auto w-full  bg-white rounded-xl py-4 px-6">
+      {currentProduct.reviews.map((review: any) => {
```

Comment:
@Kamildeeal can you add types for Review? also ReferenceComment could be a separate component